packages/node/package.json

```
@@ -20,17 +20,27 @@
     "tsc-test": "tsc -p tsconfig.test.json && tsc-alias -p tsconfig.test.json",
     "compile-test": "npm run clean && npm run tsc-test",
     "test-coverage": "npm run compile-test && c8 mocha",
-    "test": "npm run compile-test && NODE_ENV=test mocha 'dist/**/*.test.js'"
+    "test": "npm run compile-test && NODE_ENV=test mocha 'dist/**/*.test.js'",
+    "postinstall": "patch-package"
   },
   "dependencies": {
-    "@libp2p/bootstrap": "^10.0.11",
-    "@libp2p/interface-peer-id": "^2.0.2",
-    "@libp2p/tcp": "9.0.17",
-    "@waku/core": "0.0.27",
-    "@waku/relay": "0.0.10",
-    "@waku/sdk": "0.0.23",
-    "@waku/interfaces": "0.0.22",
-    "libp2p": "^1.1.2"
+    "@libp2p/bootstrap": "^10.1.2",
+    "@libp2p/identify": "^2.1.2",
+    "@libp2p/interface": "1.1.4",
+    "@libp2p/mplex": "^10.1.2",
+    "@libp2p/ping": "^1.1.2",
+    "@libp2p/tcp": "10.0.7",
+    "@libp2p/websockets": "^8.1.4",
+    "@noble/hashes": "^1.3.3",
+    "@waku/core": "0.0.32",
+    "@waku/discovery": "0.0.5",
+    "@waku/interfaces": "0.0.27",
+    "@waku/message-hash": "0.1.16",
+    "@waku/proto": "^0.0.8",
```

can we keep this as fixed If we're doing a patch over 0.0.8 ???

```suggestion
    "@waku/proto": "0.0.8",
```